cumulusci/cli/cci.py

```diff
@@ -946,6 +951,10 @@ def org_list(config, plain):
                 "username", org_config.userinfo__preferred_username
             )
             row.append(username)
+            if "expiration" in org_config.config.keys():
+                row.append(org_config.config["expiration"])
+            else:
+                row.append("None")
```

Comment:
You could simply do:

    row.append(org_config.expiration or "None")

But I think it might be slightly easier to understand if we use "never" as the text when there's no expiration instead of "none"

Reply:
I was thinking about Persistent what do you think?

Reply:
<img width="623" alt="Screen Shot 2019-11-06 at 12 34 01 PM" src="https://user-images.githubusercontent.com/28512644/68336384-2047f280-0093-11ea-9acb-f9a3caa6216e.png">

Proposed feature functionality, Unknown if its been connected to key chain pre update, Persistent if persistent org and Date time if not persistent org

Reply:
@jofsky I like that. We should probably just show the date and not the time, since expiration doesn't take effect immediately (I think there's a job that runs each night to reclaim expired orgs)

Reply:
Ok should I reformat too? I feel like I should.

Reply:
@jofsky I'd still like to see this formatted without the time.